nvflare/edge/tools/edge_recipe.py

```diff
@@ -49,6 +69,21 @@ def __init__(
 
 
 class DeviceManagerConfig:
+    """Configuration class for the device manager in federated learning.
+
+    This class configures how the device manager selects and manages devices
+    for participation in federated learning workflow.
+
+    Attributes:
+        device_selection_size: Number of devices to select for each training round.
+            Default: 100
+        min_hole_to_fill: Minimum number of model updates to wait for before
+            sampling the next batch of devices. Default: 1 (immediately dispatch the current global model)
```

Comment:
what is this for? if I set min_hole_to_fill to be the same since as device_selection_size, then that means I am synchronous training?

suggest make this more verbose to explain, something like:

```
min_hole_to_fill: int, optional (default=1)
    The minimum number of client model updates the server waits to receive 
    before aggregating and dispatching the next global model. 

    - If set to 1, the server immediately proceeds after receiving the first update.
    - Higher values cause the server to wait for more updates, potentially improving 
      aggregation quality but increasing waiting time.

    This parameter controls the trade-off between update latency and the amount of 
    client contribution used in each federated learning round.
```